src/kbe/lo.py

```diff
@@ -268,7 +268,7 @@ def localize(
                     numpy.dot, (Ciao_[k].conj().T, self.FOCK[k], Ciao_[k])
                 )
                 S_iao = reduce(numpy.dot, (Ciao_[k].conj().T, self.S[k], Ciao_[k]))
-                e_iao, v_iao = scipy.linalg.eigh(fock_iao, S_iao)
+                e_iao, _ = scipy.linalg.eigh(fock_iao, S_iao)
```

Comment:
Suggestion A: In terms of code readability (and connecting it back to equations), would it make sense to keep these as comments? (i.e. `# e_iao, v_iao` at the end of this line)

Reply:
hmm, having the `_` to explicitly tell that some tuple members are unused removes a lot of mental burden when reading the code. I am more leaning towards keeping it as is.

Reply:
I agree with `_`; I was suggesting
```suggestion
                e_iao, _ = scipy.linalg.eigh(fock_iao, S_iao) # e_iao, v_iao
```
or something similar; i don't think all of these instances deserve explicit comments, but some of them might